crates/typst/src/math/row.rs

```diff
@@ -375,16 +375,16 @@ impl Iterator for LeftRightAlternator {
 pub struct MathRunFrameBuilder {
     /// The size of the resulting frame.
     pub size: Size,
-    /// Sub frames for each row, and the positions where they should be pushed into
-    /// the resulting frame.
-    pub frames: Vec<(Frame, Point)>,
+    /// Each row: the baseline, the frame, and the positions where they should
+    /// be pushed into the resulting frame.
+    pub frames: Vec<(Abs, Frame, Point)>,
```

Comment:
I don't think this is necessary as the baseline can already be extracted from the frame